onnxmltools/convert/sklearn/operator_converters/KNeighborsClassifier.py

```diff
@@ -0,0 +1,86 @@
+# -------------------------------------------------------------------------
```

Comment:
Is it possible to merge KNNClassifier and KNNRegressor converters? Their computation graphs are quite similar. I guess we can add an if-else for creating the nodes after selecting top-k examples, so that the code for creating common structures in KNNC and KNNR can be shared.

Reply:
Done.